### PR TITLE
[PR 623 - Unsupported Tokens] Addressing comments

### PR DESCRIPTION
**Pull request by W3stside**
Addresses comments in #623 - waterfalls in.

### CODE REVIEW

File: src/custom/hooks/useRefetchPriceCallback.ts [hunk 1]
```diff
@@ -46,11 +48,41 @@ async function getQuote({
...
+
+      console.error('[useRefetchPriceCallback]::Unsupported token detected:', unsupportedTokenAddress, '- disabling.')
+
+      addUnsupportedToken({
+        chainId,
+        address: unsupportedTokenAddress.toLowerCase(),
+        dateAdded: Date.now()
+      })
+    }
+  }
```
Comment (anxolin) [inline src/custom/hooks/useRefetchPriceCallback.ts:78]: You need to add a console.error for the else too

**Review by anxolin — COMMENTED** [review]
Nice fixes!

I think this one is still unaddressed
https://github.com/gnosis/gp-swap-ui/pull/623#discussion_r631676243

**Comment by W3stside** [comment]
> #623 (comment)

@anxolin addressed in #635

**Comment by W3stside** [comment]
merging as all comments addressed and @alfetopito tested claiming it worked well for him as well.

**Pull request merged**